prime-router/src/testIntegration/resources/datatests/FHIR_to_HL7/sample_SR_1_20230302-0001.hl7

```diff
@@ -0,0 +1 @@
+MSH|^~\&|^https://simplereport.gov|^12D3456789^CLIA|||20230302213042.888+0000||ORU^R01^ORU_R01|f1265313-2bcd-4585-8482-96264c4f8abe|P|2.5.1|||||USA|UNICODE UTF-8
SFT|SimpleReport^L^^^^SimpleReport|d78782e|PRIME SimpleReport|d78782e||20230302194454+0000
PID|1||16eaac89-db3b-4983-8bfb-fa468ecabff6||Aaron Beck^Tom^^^^^L||19330511|M||2106-3^white^HL70005|123 Main Street^^Washington^IG^20008^USA||^PRN^CP^^1^248^5551234^^^^^(248) 555 1234~^NET^Internet^foo@example.com^^^^^^^^foo@example.com|||||||||N^Not Hispanic or Latino^HL70189
OBR|1|||95423-0^^LN||||||||||||^John^John|^WPN^PH^^1^248^5551234^^^^^(248) 555 1234||||||||F
OBX|1|CE|95423-0^COVID-19^LN||260373001^POSITIVE^SCT||||||F||||||||||||Third^^^^^^^^^12D3456789|1 Main St^^^IG^99551^USA
OBX|2|CE|95423-0^Flu B^LN||260373001^POSITIVE^SCT||||||F||||||||||||Third^^^^^^^^^12D3456789|1 Main St^^^IG^99551^USA
OBX|3|CE|95423-0^Flu A^LN||260373001^POSITIVE^SCT||||||F||||||||||||Third^^^^^^^^^12D3456789|1 Main St^^^IG^99551^USA
OBX|4|CE|95419-8^Has symptoms related to condition of interest^LN||Y^Yes^HL70136||||||F
OBX|5|DT|11368-8^Illness or injury onset date and time^LN||20230301||||||F
SPM|1|||258500001^Nasopharyngeal Swab^SCT

```

Comment:
Each of the segments should be on a new line

Reply:
This is a github parsing issue. When I pull the pr down locally, intelij parses it properly. Please ignore